dm/master/coordinator/coordinator.go

```diff
@@ -46,11 +43,11 @@ type Coordinator struct {
 	// upstream(source-id) -> worker
 	upstreams map[string]*Worker
 
-	// upstream(address) -> config
+	// upstream(address) -> source-id
```

Comment:
So, Is it not _worker_ to _config_ now? Giving it a meaningful name?

Reply:
Yes, the relationship is address -> source id now. I will change it